requirements.txt

```diff
@@ -1 +1 @@
-openpyxl>=2.6.1
+openpyxl==2.6.1
```

Comment:
should this still be >= so that it does not break if HQ updates the openpyxl version it uses

Reply:
we dont need this file any more yes?